examples/seismic/skew_self_adjoint/wavesolver.py

```diff
@@ -137,7 +138,7 @@ def adjoint(self, rec, src=None, b=None, v=None, wOverQ=None, u=None,
             The time-constant buoyancy.
         v : Function or float, optional, defaults to v at construction
             The time-constant velocity.
-        wOverQ : Function or float, optional, defaults to wOverQ at construction
+        damp : Function or float, optional, defaults to damp at construction
```

Comment:
the "defaults ..." goes in the description (line below)

Reply:
fixed